index.html

```diff
@@ -0,0 +1,853 @@
+          <div class="flex-txt">
+            <h3>Why you always need to show your product's 'humanness'</h3>
+            <p>Boris Veldhuijzen van Zanten 16 minutes ago</p>
+          </div>
+        </div>
+        <div class="grid1 sec2-media">
+          <img class="flex-img" src="images/grid1.png" alt="pic here" />
+          <div class="flex-txt">
+            <h3>Why you always need to show your product's 'humanness'</h3>
+            <p>Boris Veldhuijzen van Zanten 16 minutes ago</p>
+          </div>
+        </div>
+        <div class="grid1 sec2-media">
+          <img class="flex-img" src="images/grid2.png" alt="pic here" />
+          <div class="flex-txt">
+            <h3>Why you always need to show your product's 'humanness'</h3>
+            <p>Boris Veldhuijzen van Zanten 16 minutes ago</p>
+          </div>
+        </div>
+        <div class="grid1 sec2-media">
+          <img class="flex-img" src="images/grid3.png" alt="pic here" />
+          <div class="flex-txt">
+            <h3>Why you always need to show your product's 'humanness'</h3>
+            <p>Boris Veldhuijzen van Zanten 16 minutes ago</p>
+          </div>
+        </div>
+        <div class="grid1 sec2-media">
+          <img class="flex-img" src="images/grid4.png" alt="pic here" />
+          <div class="flex-txt">
+            <h3>Why you always need to show your product's 'humanness'</h3>
+            <p>Boris Veldhuijzen van Zanten 16 minutes ago</p>
+          </div>
+        </div>
+        <div class="grid1 sec2-media">
+          <img class="flex-img" src="images/grid5.png" alt="pic here" />
+          <div class="flex-txt">
+            <h3>Why you always need to show your product's 'humanness'</h3>
+            <p>Boris Veldhuijzen van Zanten 16 minutes ago</p>
+          </div>
+        </div>
+        <div class="grid1 sec2-media">
+          <img class="flex-img" src="images/grid6.png" alt="pic here" />
+          <div class="flex-txt">
+            <h3>Why you always need to show your product's 'humanness'</h3>
+            <p>Boris Veldhuijzen van Zanten 16 minutes ago</p>
+          </div>
+        </div>
+        <div class="grid1 sec2-media">
+          <img class="flex-img" src="images/grid7.png" alt="pic here" />
+          <div class="flex-txt">
+            <h3>Why you always need to show your product's 'humanness'</h3>
+            <p>Boris Veldhuijzen van Zanten 16 minutes ago</p>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="section-3">
+      <div class="section-3-top flex">
+        <h1>
+          <a href="#">
+            Latest Funding rounds on index.co <i class="fas fa-angle-right"></i
+          ></a>
+        </h1>
+        <div class="sec-3-top-parts flex">
+          <a href="#">Add Info</a>
+          <a class="section-3-top-a" href="#">Get updates by email</a>
+          <a href="#">See More</a>
+        </div>
+      </div>
```

Comment:
- [ ] Notice that you have the See more, Get updates, Add info links in the same row and in the original it is displaying below the title, please emulate the original. (This is an easy fix, you can use `flex-direction: column;` in the medium size of your .section-3-top flex to change the way they are positioned)
![image](https://user-images.githubusercontent.com/53256139/92090109-a5c03000-ed94-11ea-9b89-e593ea344e3e.png)